client/src/components/HostStats.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { query } from '../utils';
+
+const HostStatsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 8px;
+  @media (min-width: ${query.medium}) {
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-bottom: 12px;
+  }
+`;
+
+const StatText = styled.p`
+  font-size: 1.6rem;
+  margin: 0 0 16px 0;
+  padding: 0 12px;
+`;
+
+const HostStats = ({ isSuperhost, isVerified, reviews }) => {
+  return (
+    <HostStatsContainer>
+      {reviews && <StatText>⭐️ {reviews} Reviews</StatText>}
```

Comment:
Woah!!!! Did not know you could do emojis like this!!!! I'm so happy

Reply:
Does this shorthand mean it won't show if reviews is false-y?

Reply:
@rgagnon2 Yep!